sklearn/datasets/tests/test_base.py

```diff
@@ -202,6 +203,17 @@ def test_load_iris():
     check_return_X_y(res, partial(load_iris))
 
 
+def test_load_iris_as_frame():
+    try:
+        data_frame, target_series = load_iris(as_frame=True)
```

Comment:
I don't get why this would raise SkipTest. Why not use importorskip?

Reply:
while looking at other fragments of the code i though this was the way to skip the pandas test. let me check importorskip.

Reply:
pytest is relatively new to us. But it provides importorskip for this kind of application.